src/Config/Config.py

```diff
@@ -118,4 +118,5 @@ def get_user(self):
             config = json.load(config_file)
         if(len(config["user"]["firstname"]) == 0):
             raise UserNotSet()
-        return config["user"]
+        for info in config["user"]:
+            print(info + " : " + str(config["user"][info]))
```

Comment:
Earlier approach was correct. Receive user object from Config.py and print it in config.py